genestack_client/utils.py

```diff
@@ -19,6 +19,14 @@ def isatty():
         return False
 
 
+class GenestackArgumentParser(argparse.ArgumentParser):
+    def parse_known_args(self, args=None, namespace=None):
+        args, argv = super(GenestackArgumentParser, self).parse_known_args(args, namespace)
+        if not args.user and args.pwd:
```

Comment:
Let's write `args.pwd and not args.user`, that way it's harder to misunderstand it as `not (args.user and args.pwd)`.